### PR TITLE
Add Hello Worlds M1 pair project

### PR DESCRIPTION
**Pull request by letakeane**
**Description of changes made**
- I created a new M1 pair project
- I created the spec for the project
- I created the boilerplate repo for the project (found [here](https://github.com/turingschool-examples/hello-worlds-boilerplate))
- I updated the project index page to add a link to the spec

**Motivation for any changes**
- ticket asked me to
- Need a project more strongly differentiated from RomCom/Hang in There for repeaters

**Questions that you have for a reviewer**
- This project only has 3 iterations, compared to RomCom/Hang in There's 4 - is it too simple?
- Is the data model robust enough? 
  - As of right now, it contains: all the questions, the current question, and the user's scores
- Do the optional extensions suggestions seem reasonable?

Closes https://github.com/orgs/turingschool/projects/6/views/1?pane=issue&itemId=25265636

### CODE REVIEW

File: projects/module-1/hello-worlds.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
+---
+title: Hello Worlds - Pair
+---
+
+## Learning Goals
+
+* Write clean, DRY JavaScript
+  * Build out functionality using functions that show trends toward SRP
+  * Manipulate the page after it has loaded adding, removing, and updating elements on the DOM
```
Comment (robbiejaeger) [inline projects/module-1/hello-worlds.md]: Not sure what this sentence is saying. Is it meant to be two parts?

"Manipulate the page after it has loaded - adding, removing, and updating elements on the DOM"

File: projects/module-1/hello-worlds.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+We're turning this into a game about language! Our users will guess what language the phrase "Hello world!" appears in, and we'll keep track of their guesses.
+
+## Resources 
+
+<section class="answer">
+### Helpful Hints
+
+<section class="answer">
+### What's the deal with deploying?
```
Comment (robbiejaeger) [inline projects/module-1/hello-worlds.md]: This section is at the beginning of the project spec, which could make a student think that it is critical for their project form the start, but I see deployment as a nice to have at the end. Is this something that would make send to be moved toward the end of the spec?

File: projects/module-1/hello-worlds.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+To begin, choose **ONE** partner to do the following:
+
+1. Fork the repository found here: [https://github.com/turingschool-examples/hello-worlds-boilerplate/](https://github.com/turingschool-examples/hello-worlds-boilerplate/).
+2. Clone down your new, forked repo.  While cloning, name it what you want your project to be named, should not include "boilerplate". `git clone <url> <newNameYouWantItToHave>`
+3. `cd` into the repository
+4. Open it in your text editor
+5. View the project in the browser by running `open index.html` in your terminal
+6. Add all project partners and your assigned instructor as collaborators on the repository
+
+Then, as a team:
```
Comment (robbiejaeger) [inline projects/module-1/hello-worlds.md:75]: Do you want to add a note here that everyone else should clone down the repo at this point?

File: projects/module-1/hello-worlds.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+- When the page loads, we should see a game screen with the phrase, 3 guess options, and player guess scores displayed.
+</section>
+
+<section class="answer">
+### Iteration 1 - Guessing
+
+![after first guess](../../assets/images/projects/hello-worlds/after-guess.png)
+
+- When a user clicks one of the guess buttons, we should see the answer
+- When a user clicks ones of the guess buttons, we should see feedback about their answer
```
Comment (robbiejaeger) [inline projects/module-1/hello-worlds.md]: Did you mean "one"?

File: projects/module-1/hello-worlds.md [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+- When a user clicks one of the guess buttons, we should see that the guess buttons are disabled, preventing the user from making a different guess for the same question
+- When a user clicks one of the guess buttons, we should see that the Guess Again reset button is enabled.
+
+<section class="note">
+### Hint!
+
+Go check out the HTML and CSS files to see how the reset buttonis being disabled in the first place.
```
Comment (robbiejaeger) [inline projects/module-1/hello-worlds.md]: Space for "button is"?

File: projects/module-1/hello-worlds.md [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+This means, we DO NOT want to see:
+
+* Code that completes iterations but is sloppy
+* One or both team members do not understand every single line of code
+* One or both team members skips the problem solving process (pseudocoding, talking out the problem, articulating, planning) in the pursuit of completing functionality
+
+**Well-refactored, thoughtful code is better than sloppy extra features.**
+
+💫ON TRACK💫 can look like:
+- All of the expectations of iterations 0-4 are complete without bugs
```
Comment (robbiejaeger) [inline projects/module-1/hello-worlds.md]: Is this "0-3" for this project?

**Comment by robbiejaeger** [comment]
I left a few comments - it looks like a really fun project that isn't just a standard trivia game. Cool project, Leta!

File: projects/module-1/hello-worlds.md (hunk 1 above)
Comment (letakeane) [reply projects/module-1/hello-worlds.md]: Yes - I'll add clarifying punctuation! I just copy pasted it from another M1 project :sweat_smile:

File: projects/module-1/hello-worlds.md (hunk 2 above)
Comment (letakeane) [reply projects/module-1/hello-worlds.md]: Great idea! I will shift this down below the extensions section.

File: projects/module-1/hello-worlds.md (hunk 3 above)
Comment (letakeane) [reply projects/module-1/hello-worlds.md:75]: Yes, great note! I'll add it.

File: projects/module-1/hello-worlds.md (hunk 4 above)
Comment (letakeane) [reply projects/module-1/hello-worlds.md]: Yes I did, haha. Thanks!

**Comment by letakeane** [comment]
Made changes based on Robbie's comments! I'm going to merge into development.

**Pull request merged**